app/src/main/AndroidManifest.xml

```diff
@@ -8,24 +9,28 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/EditTextPrefTheme">
+        android:theme="@style/EditTextPrefTheme"
+        tools:ignore="GoogleAppIndexingWarning">
+        <activity android:name=".SplashActivity"
+            android:theme="@style/AppTheme.SplashScreen">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
         <activity
             android:name=".ProgramModeActivity"
-            android:parentActivityName=".MainActivity"/>
+            android:parentActivityName=".MainActivity" />
```

Comment:
Why this change? Please remove.

Reply:
I just reformatted the code by pressing CTRL+ALT+L nothing else.

Reply:
@jddeep Reformatting is necessary and don't worry this won't cause merge conflicts.